aspnetcore/blazor/fundamentals/index.md

```diff
@@ -69,6 +69,31 @@ The preceding `Counter` component:
 * Renders the current count with `@currentCount`. `currentCount` is an integer variable defined in the C# code of the `@code` block.
 * Displays a button to trigger the `IncrementCount` method, which is also found in the `@code` block and increases the value of the `currentCount` variable.
 
+:::moniker range=">= aspnetcore-8.0"
+
+## Render modes
+
+Articles in the *Fundamentals* node make reference to the concept of *render modes*. This subject is covered in detail in the <xref:blazor/components/render-modes> article in the *Components* node, which appears after the *Fundamentals* node of articles.
+
+For the early references in this node of articles to render mode concepts, merely note the following at this time:
+
+Every component in a Blazor Web App adopts a *render mode* to determine the hosting model that it uses, where it's rendered, and whether or not it's rendered statically on the server, rendered with for user interactivity on the server, or rendered for user interactivity on the client (usually with prerendering on the server).
```

Comment:
From what I've observed over the years, is that a lot of new developers have a hard time understanding what client / server rendering is. I think a diagram or a short explanation of the difference is worth adding here. Something, showing that `client-side` rendering is the renderieng, handled by the Blazor WebAssembly runtime running on the client (browser). Thus, there is no HTML being sent to the client from the server for the generated UI. Whereas, server rendering handles rendering of components on the server and sends the generated HTML to the client over the wire. I know this may seem very basic, and is probably already covered somewhere, but it's worth to put a nice writeup or reference about this to help the readers.

Reply:
The same applies to the concept of `interactivity`. I think it's worth to explain that too (not here), as I've seen several Reddit comments recently where people weren't sure what it means.

Reply:
I'll work on this on Sunday morning ... when Merlin the 😸 wakes me up at 5am again 😩.